CHANGELOG.md

```diff
@@ -3,6 +3,37 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [6.0.0] - TBD
+### Added
+- Per-schema option to send S3 data notifications to an SQS queue.  See `enable_data_event_queue` in the [apiary_managed_schemas](VARIABLES.md#apiary_managed_schemas) section of [VARIABLES.md](VARIABLES.md)
```

Comment:
can we use `enable_data_events` instead of `enable_data_event_queue` as queue is only created once per datalake  instead of per schems.

Reply:
We already have `var.enable_data_events` for creating data notifications to SNS topics.  Are you suggesting we use the same named variable in the schema info map to trigger that schema to use SQS instead of SNS?  If that is what you are saying, won't that be confusing?

Reply:
Yes, I think `enable_data_events` still makesense, `enable_data_event_queue` implies we are going to create a queue per schema.
or else,what do you think of `enable_data_events_sqs`

Reply:
Changed to `enable_data_event_queue`.

Reply:
Haha, I meant to change `enable_data_event_queue` to `enable_data_events_sqs` based on your PR comment, but through the magic of copy/paste errors, I changed it to exactly what it already was. :slightly_smiling_face: 

Fixed now.